src/Entity/Transformer.php

```diff
@@ -216,30 +224,6 @@ public function getChildren(PageInterface $page)
         return $this->children[$page->getId()];
     }
 
-    /**
-     * @return array
-     */
-    private function fixPageContent(array $content)
-    {
-        if (!\array_key_exists('title', $content)) {
```

Comment:
Did you check if we are using title in other place, or it was just forget here?

Reply:
I dont see the point on those fix content methods. the isset method already makes sure it exists and it is not null.